projects/renku_data_service/Dockerfile

```diff
@@ -33,5 +33,5 @@ RUN apt-get update && apt-get install -y \
 USER $USER_UID:$USER_GID
 WORKDIR /app
 COPY --from=builder /app/env ./env
-ENTRYPOINT ["tini", "-g", "--"]
-CMD ["tini", "-g", "--", "env/bin/python", "-m", "renku_data_services.data_api.main", "--fast"]
+ENTRYPOINT ["tini", "-g", "--", "env/bin/python", "-m", "renku_data_services.data_api.main"]
+CMD ["--single-process"]
```

Comment:
I think we should just run without an argument:
> By default, Sanic runs a main process and a single worker process (see [worker manager](https://sanic.dev/en/guide/deployment/manager.html) for more details).
from [here](https://sanic.dev/en/guide/deployment/running.html#workers)

Because on single process mode, it says:
> ou will also not have access to any features that require processes (auto-reload, the inspector, etc).
and I don't know if I'm comfortable with not having "etc."...

Reply:
Oh cool I did not know that. I will remove that. I did not know about these limitations.